util/webhook/webhook.go

```diff
@@ -244,14 +273,66 @@ func affectedRevisionInfo(payloadIf interface{}) (webURLs []string, revision str
 	return webURLs, revision, change, touchedHead, changedFiles
 }
 
+func fetchDiffstatFromBitbucket(workspace, repoSlug, spec, accessToken string) ([]string, error) {
+	// Getting the files changed from diff API:
+	// https://developer.atlassian.com/cloud/bitbucket/rest/api-group-commits/#api-repositories-workspace-repo-slug-diffstat-spec-get
```

Comment:
```suggestion
// fetchDiffstatFromBitbucket gets the files changed from diff API - https://developer.atlassian.com/cloud/bitbucket/rest/api-group-commits/#api-repositories-workspace-repo-slug-diffstat-spec-get
```
Please add this as a function comment (at the top of function)